internal/pkg/endpoints/kubernetes.go

```
@@ -178,17 +178,14 @@ func objectTargets(object metav1.Object) []Target {
 func serviceTarget(s *apiv1.Service, port, path string) Target {
 	lbls := labels.Set{}
 	hostname := fmt.Sprintf("%s.%s.svc", s.Name, s.Namespace)
-	addr := url.URL{
-		Scheme: "http",
-		Host:   net.JoinHostPort(hostname, port),
-		Path:   path,
-	}
+	hostAndPort := net.JoinHostPort(hostname, port)
+	addr, _ := url.Parse(fmt.Sprintf("http://%s%s", hostAndPort, path))
```

why are you ignoring this error?